docs/pages/versions/unversioned/react-native/safeareaview.md

```diff
@@ -25,7 +27,7 @@ const App = () => {
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
+    flex: 1, // required, so the SafeAreaView has a height greater than zero
```

Comment:
```suggestion
    * @info Remember to set correct style, so the SafeAreaView fills the entire available space.*/flex: 1,/* @end */
```

Let's use highlight and tooltip instead of plain comment.